Author: Alexander Gerasiov <gq@debian.org>
Description: Call setlocale(LC_ALL, "") on init.
 This fix various issues with messages and encodings.
Updated: 2015-03-02

--- a/minidlna.c
+++ b/minidlna.c
@@ -479,8 +479,7 @@
 static void init_nls(void)
 {
 #ifdef ENABLE_NLS
-	setlocale(LC_MESSAGES, "");
-	setlocale(LC_CTYPE, "en_US.utf8");
+	setlocale(LC_ALL, "");
 	DPRINTF(E_DEBUG, L_GENERAL, "Using locale dir %s\n", bindtextdomain("minidlna", getenv("TEXTDOMAINDIR")));
 	textdomain("minidlna");
 #endif
